### PR TITLE
kepprfint

### PR DESCRIPTION
**Pull request by rpoleski**
The kepprf() and kepprfphot() functions had significant blocks of code that were almost the same. I've moved this blocks to a new function and removed some inconsistencies (e.g. `prfWeight[i] = 1.0e-6` vs. `1.0e6`). Minor improvements in the code were made.

### CODE REVIEW

**Comment by mirca** [comment]
Hi @rpoleski, thank you for organizing that! I would suggest that you move the implementation of the function ``read_and_interpolate_prf()`` to the module ``kepfunc`` rather than creating another module.

Other than that, I think it's ready to merge! 👍

**Comment by barentsen** [comment]
Thanks @rpoleski!  If you make the change suggested by @mirca, please also add your name to `AUTHORS.rst`.

**Comment by rpoleski** [comment]
Ok, I've made the requested change. I've also improved the interpolation - it's now using centre of TPF file instead of the corner. I've also added my name to authors file. Thanks.

**Comment by barentsen** [comment]
Thanks @rpoleski!

File: pyke/kepfunc.py [hunk 1]
```diff
@@ -775,3 +777,116 @@ def inv_normal_cummulative_function(p):
...
+    """
+    n_hdu = 5
+    minimum_prf_weight = 1.e-6
+
+    # determine suitable PRF calibration file
+    if int(module) < 10:
+        prefix = 'kplr0'
+    else:
+        prefix = 'kplr'
+    prfglob = os.path.join(prfdir, prefix + module + '.' + output + '*_prf.fits')
```
Comment (mirca) [inline pyke/kepfunc.py]: Let's make sure we add ``import os`` and ``import glob`` at the top of the ``kepfunc.py`` module.

**Comment by rpoleski** [comment]
@mirca , it's done.

**Comment by mirca** [comment]
@rpoleski thanks! Merged!

**Pull request merged**